src/tests/fuzz/fuzz_sigpcap_aware.c

```diff
@@ -157,7 +157,7 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
     // loop over packets
     r = FPC_next(&pkts, &header, &pkt);
     p = PacketGetFromAlloc();
-    if (header.ts.tv_sec >= INT_MAX - 3600) {
+    if (v header.ts.tv_sec >= INT_MAX - 3600) {
```

Comment:
this looks like a paste error

while at it, could you add a check that makes sure tv_usec >= 0 (for each of these pcap targets)? I've seen rust overflow panics still due to negative values

Reply:
Ok thanks